animations/lib/src/basics/08_fade_transition.dart

```diff
@@ -65,18 +62,13 @@ class _FadeTransitionDemoState extends State<FadeTransitionDemo>
         ),
       ),
       floatingActionButtonLocation: FloatingActionButtonLocation.centerFloat,
-      floatingActionButton: FloatingActionButton.extended(
+      floatingActionButton: RaisedButton(
```

Comment:
This should probably be a sibling of the star widget in a `Column` instead of the `floatingActionButton` widget.